src/lib/wdi5-control.ts

```diff
@@ -151,8 +153,12 @@ export class WDI5Control {
      * @returns
      */
     async renewWebElement(id: string = this._domId) {
-        this._webdriverRepresentation = await this._browserInstance.$(`//*[@id="${id}"]`)
-        return this._webdriverRepresentation
+        if (this._domId) {
+            this._webdriverRepresentation = await this._browserInstance.$(`//*[@id="${id}"]`)
+            return this._webdriverRepresentation
+        } else {
+            throw Error("control could not be found")
```

Comment:
could some sort of an identifier be added to the error message to provide an even better aka more fine-tuned feedback?

Reply:
same as above